src/main/java/com/minecolonies/coremod/colony/ColonyList.java

```diff
@@ -15,15 +15,14 @@
 import java.util.stream.StreamSupport;
 
 //TODO extend list/collection
+
 /**
  * Data structure for storing colonies, optimized for performance.
  *
- * @author Colton
- *
  * @param <T> Type of IColony (Colony or ColonyView)
+ * @author Colton
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') At-clauses have to appear in the order '[@author, @version, @param, @return, @throws, @exception, @see, @since, @serial, @serialField, @serialData, @deprecated]'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.javadoc.AtclauseOrderCheck)